test/client/fixtures/sandbox/event/event-test.js

```diff
@@ -301,3 +302,44 @@ test('SVGElement.dispatchEvent should be overriden (GH-614)', function () {
 
     svg.parentNode.removeChild(svg);
 });
+
+test('should not wrap invalid event handlers (GH-1251)', function () {
+    var handlers        = [void 0, 1, null, 'str', {}];
+    var invalidHandlers = [];
+    var errorText       = null;
+
+    if ('Symbol' in window)
+        handlers.push(Symbol('foo'));
+
+    // NOTE: some handlers in some browsers throw the "Invalid argument" error
```

Comment:
// NOTE:  on adding some type of handlers an "Invalid argument" error can be raised